### PR TITLE
Atmel AT86RF230 in avr-rcb: added missing macros and initialisations 

### PR DESCRIPTION
**Pull request by Tom435**
I added important modules in the Makefile.avr-rcb as well as some macros in contiki-conf.h and Makefile.avr-rcb to be able to compile applications for the avr-rcb.
To use the wireless interface it is necessary to initialize the NETSTACK_* protocols in /contiki-rcb-main.c.

### CODE REVIEW

File: platform/avr-rcb/contiki-rcb-main.c [hunk 1]
```diff
@@ -109,21 +109,22 @@ init_lowlevel(void)
...
-  uint8_t error;
 
   PROCESS_BEGIN();
   
-  if((error = icmp6_new(NULL)) == 0) {
     while(1) {
       PROCESS_YIELD();
       
-#if NETSTACK_CONF_WITH_IPV6	  
+#if UIP_CONF_IPV6
```
Comment (nvt) [inline platform/avr-rcb/contiki-rcb-main.c]: NETSTACK_CONF_WITH_IPV6 is the normal way to check whether IPv6 is in use. Why was this changed?

**Comment by Tom435** [comment]
You are right, that´s nonsense. I accidentally changed that macro because of a copy and paste mistake.

File: platform/avr-rcb/contiki-rcb-main.c (hunk 1 above)
Comment (Tom435) [inline platform/avr-rcb/contiki-rcb-main.c]: You are right, that´s nonsense. I accidentally changed that macro because of a copy and paste mistake.

**Comment by nvt** [comment]
:+1:
